docs/content/guides/columns/column-filter.md

```diff
@@ -155,765 +450,2860 @@ import 'handsontable/dist/handsontable.full.min.css';
-            )}
-          </select>
-        </div>
-
-        <div id="filterSelect">
-          <div className="controllers">
-            <div>
-              <label><input type="checkbox" onChange={(event) => toggleSelectAll(event)} id="filtersSelectAll"
-                defaultChecked="checked"/> (Select all)</label>
-            </div>
-          </div>
-          <div className="items">
-            {rowEntries.map(
-              (rowEntry, index) => <React.Fragment key={`${rowEntry}-${index}`}>
-                <label><input type="checkbox" onChange={() => selectEntry(rowEntry)}
-                  checked={selectedRowEntries.includes(rowEntry)}/>{rowEntry}</label><br/>
-              </React.Fragment>
-            )}
-          </div>
-        </div>
-
-        <div className="buttons controls">
-          <button onClick={() => applyFilter()} className="apply">Apply filter</button>
-          <button onClick={() => clearFilter()} className="clear">Clear filter</button>
-        </div>
+      <div className="controls">
+        <button onClick={filterBelow200}>Filter for items below $200</button>
+        <br />
+        <br />
+        <button onClick={filterAbove200}>Filter for items above $200</button>
+        <br />
+        <br />
+        <button onClick={clearAllFilters}>Clear all filters</button>
       </div>
-
     </>
   );
 };
 
 /* start:skip-in-preview */
-ReactDOM.render(<ExampleComponent />, document.getElementById('example4'));
+ReactDOM.render(<App />, document.getElementById('exampleFilterThroughAPI1'));
 /* end:skip-in-preview */
 ```
+
 :::
+
 :::
 
-## Related API reference
+## Import the filtering module
+
+You can reduce the size of your bundle by importing and registering only the
+[modules](@/guides/tools-and-building/modules.md) that you need.
+
+To use filtering, you need only the following modules:
+
+- The [base module](@/guides/tools-and-building/modules.md#import-the-base-module)
+- The [`DropdownMenu`](@/api/dropdownMenu.md) module
+- The [`Filters`](@/api/filters.md) module
+
+```js
+// import the base module
+import Handsontable from 'handsontable/base';
+
+// import Handsontable's CSS
+import 'handsontable/dist/handsontable.full.min.css';
+
+// import the filtering plugins
+import { registerPlugin, DropdownMenu, Filters } from 'handsontable/plugins';
+
+// register the filtering plugins
+registerPlugin(DropdownMenu);
+registerPlugin(Filters);
+```
+
+## Known limitations
+
+At the moment, filtering comes with the following limitations:
+
+- You can't use filtering with [nested data structures](@/guides/rows/row-parent-child.md)
+  ([`NestedRows`](@/api/nestedRows.md)).
+- You can't add custom filter conditions.
+- You can't control the **Filter by value** list. The list of values is generated automatically and
+  there's no supported way of modifying it.
+- There's no built-in protection against filtering passwords.
```

Comment:
What do you mean by that? It's unclear to me, even though I know a lot about issues with presenting passwords in the filters menu.